### PR TITLE
fix: corrected upload_fmu uri

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**
This PR corrected the route for FMU imports.

### CODE REVIEW

File: modelon/impact/client/sal/project.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def fmu_upload(
         url = (
             self._base_uri
             / f"api/workspaces/{workspace_id}/projects/{project_id}/content/"
```
Comment (efredriksson-modelon) [inline modelon/impact/client/sal/project.py:89]: And it ´does not contain workspaces/{workspace_id} anymore? That is the bigger change.

**Review by efredriksson-modelon — CHANGES_REQUESTED** [review]
I think there are two issues with the URI :smile:

**Comment by abhilash-kumar-nair** [comment]
Hej @efredriksson-modelon , @iakovn , Could you please take a look at the fmu import changes? Also corrected the ContentType enum which was outdated.

**Review by iakovn — APPROVED** [review]
Looking good! 👍

File: modelon/impact/client/entities/model.py [hunk 2]
```diff
@@ -276,3 +281,112 @@ def new_experiment_definition(
...
+        return FMUImportOperation[Model](
+            resp['data']['location'],
+            self._workspace_id,
+            self._project_id,
+            self._sal,
+            Model.from_operation,
+        )
+
+    @classmethod
+    def from_operation(cls, operation: BaseOperation[Model], **kwargs: Any) -> Model:
```
Comment (efredriksson-modelon) [inline modelon/impact/client/entities/model.py:390]: Should this method be private?

**Review by efredriksson-modelon — APPROVED** [review]
Nice work :+1:

File: modelon/impact/client/entities/model.py (hunk 2 above)
Comment (abhilash-kumar-nair) [reply modelon/impact/client/entities/model.py:390]: Yes I think it does. I will push another PR doing this for all the from_operation class methods. Thanks! :)

**Pull request merged**